firestore-bigquery-export/firestore-bigquery-change-tracker/package.json

```diff
@@ -5,7 +5,7 @@
     "url": "github.com/firebase/extensions.git",
     "directory": "firestore-bigquery-export/firestore-bigquery-change-tracker"
   },
-  "version": "1.1.20",
+  "version": "1.1.22",
```

Comment:
bumping straight to `1.1.22`? is there a `1.1.21`?

Reply:
Yeah, there's a `1.1.21`, I re-ran the lock file on the change tracker which causes all sorts of Typescript issues.

After publishing, I needed to re-run and publish with the original lock file and just a version bump.